ts/components/wallet/payment/PickNotAvailablePaymentMethodListItem.tsx

```diff
@@ -138,16 +138,7 @@ const extractInfoFromPaymentMethod = (
         snapPoint: 360
       };
     case "Privative":
-      return {
-        logo: paymentMethod.icon,
-        title: paymentMethod.caption,
-        description: getPickPaymentMethodDescription(
-          paymentMethod,
-          nameSurname
-        ),
-        bottomSheetTitle: unacceptedBottomSheetTitle(),
-        bottomSheetBody: unacceptedBottomSheetBody()
-      };
+      return undefined as unknown as PaymentMethodInformation;
```

Comment:
This is not correct: `undefined` is not an `unknown` which is not a defined `PaymentMethodInformation`. We should remove `Privative` kind from `PaymentMethod` instead.